From 8128a7c1d15c2a1e2c1cbbc5316c4a610808a775 Mon Sep 17 00:00:00 2001
Message-ID: <8128a7c1d15c2a1e2c1cbbc5316c4a610808a775.1705743628.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705743627.git.sharlatanus@gmail.com>
References: <cover.1705743627.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 16 Jan 2024 23:30:57 +0000
Subject: [PATCH 17/38] gnu: go-github-com-aperturerobotics-jacobsa-crypto:
 Move to (gnu packages golang-crypto).

* gnu/packages/golang.scm (go-github-com-aperturerobotics-jacobsa-crypto):
Move from here ...
* gnu/packages/golang-crypto.scm: ... to here.

* gnu/packages/file-systems.scm: Add (gnu packages golang-crypto) to
used modules.

Change-Id: I9562a3a3054b7d361758c5e5984891970f191ada
---
 gnu/packages/file-systems.scm  |  1 +
 gnu/packages/golang-crypto.scm | 31 +++++++++++++++++++++++++++++++
 gnu/packages/golang.scm        | 31 -------------------------------
 3 files changed, 32 insertions(+), 31 deletions(-)

diff --git a/gnu/packages/file-systems.scm b/gnu/packages/file-systems.scm
index 8cac1eae9c..4a0976ad84 100644
--- a/gnu/packages/file-systems.scm
+++ b/gnu/packages/file-systems.scm
@@ -72,6 +72,7 @@ (define-module (gnu packages file-systems)
   #:use-module (gnu packages gnupg)
   #:use-module (gnu packages golang)
   #:use-module (gnu packages golang-check)
+  #:use-module (gnu packages golang-crypto)
   #:use-module (gnu packages guile)
   #:use-module (gnu packages jemalloc)
   #:use-module (gnu packages kerberos)
diff --git a/gnu/packages/golang-crypto.scm b/gnu/packages/golang-crypto.scm
index 860ee82727..3bcf86673d 100644
--- a/gnu/packages/golang-crypto.scm
+++ b/gnu/packages/golang-crypto.scm
@@ -5,6 +5,7 @@
 ;;; Copyright © 2022, 2023 Nicolas Graves <ngraves@ngraves.fr>
 ;;; Copyright © 2023 Artyom V. Poptsov <poptsov.artyom@gmail.com>
 ;;; Copyright © 2023 Clément Lassieur <clement@lassieur.org>
+;;; Copyright © 2023 Felix Lechner <felix.lechner@lease-up.com>
 ;;;
 ;;; This file is part of GNU Guix.
 ;;;
@@ -92,6 +93,36 @@ (define-public go-github-com-aead-chacha20
 is standardized in RFC 7539.")
       (license license:expat))))
 
+(define-public go-github-com-aperturerobotics-jacobsa-crypto
+  (let ((commit "b1eb679742a8deed015a4406384eea6bd985d08a")
+        (revision "0"))
+    (package
+      (name "go-github-com-aperturerobotics-jacobsa-crypto")
+      (version (git-version "1.0.1" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/aperturerobotics/jacobsa-crypto")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32 "16dxigj8m6q18xqsy72iq287rh4fw0y0b9yqlw0qkclb8379n1z2"))))
+      (build-system go-build-system)
+      (arguments
+       (list #:import-path "github.com/aperturerobotics/jacobsa-crypto"
+             ;; Source-only package.
+             #:tests? #f
+             #:phases
+             #~(modify-phases %standard-phases
+                 ;; Source-only package.
+                 (delete 'build))))
+      (home-page "https://github.com/aperturerobotics/jacobsa-crypto")
+      (synopsis "Cryptography missing from the Go standard library")
+      (description "This repository contains Go packages related to
+cryptographic standards that are not included in the Go standard library.")
+      (license license:asl2.0))))
+
 (define-public go-github-com-gaukas-godicttls
   (package
     (name "go-github-com-gaukas-godicttls")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index c17dc5b16d..37edfd7eb0 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -1420,37 +1420,6 @@ (define-public go-github-com-hanwen-go-fuse-v2
        "This is a repository containing Go bindings for writing FUSE file systems.")
       (license license:bsd-3))))
 
-(define-public go-github-com-aperturerobotics-jacobsa-crypto
-  (let ((commit "b1eb679742a8deed015a4406384eea6bd985d08a")
-        (revision "0"))
-    (package
-      (name "go-github-com-aperturerobotics-jacobsa-crypto")
-      (version (git-version "1.0.1" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                      (url "https://github.com/aperturerobotics/jacobsa-crypto")
-                      (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "16dxigj8m6q18xqsy72iq287rh4fw0y0b9yqlw0qkclb8379n1z2"))))
-      (build-system go-build-system)
-      (arguments
-       (list #:import-path "github.com/aperturerobotics/jacobsa-crypto"
-             ;; Source-only package.
-             #:tests? #f
-             #:phases
-             #~(modify-phases %standard-phases
-                 ;; Source-only package.
-                 (delete 'build))))
-      (home-page "https://github.com/aperturerobotics/jacobsa-crypto")
-      (synopsis "Cryptography missing from the Go standard library")
-      (description
-       "This repository contains Go packages related to cryptographic standards that are
-not included in the Go standard library.")
-      (license license:asl2.0))))
-
 (define-public go-github-com-jacobsa-reqtrace
   (let ((commit "245c9e0234cb2ad542483a336324e982f1a22934")
         (revision "0"))
-- 
2.41.0

